### PR TITLE
Allow spaces in marks, [[mark links]], M action

### PR DESCRIPTION
**Pull request by platers**
Can link to marks with [[im a mark]] format.

New action `M` to quickly mark a row with its own contents.

### CODE REVIEW

File: src/plugins/marks/index.tsx [hunk 1]
```diff
@@ -584,6 +602,37 @@ export class MarksPlugin {
 
     return null;
   }
+
```
Comment (WuTheFWasThat) [inline src/plugins/marks/index.tsx:605]: should it be just combined into a single function that takes in the cursor column?

Comment (WuTheFWasThat) [reply src/plugins/marks/index.tsx:605]: this way is also fine if you think they might be used in isolation

**Review by WuTheFWasThat — COMMENTED** [review]
overall lgtm

File: src/plugins/marks/index.tsx (hunk 1 above)
Comment (platers) [reply src/plugins/marks/index.tsx:605]: I think it might be useful this way later. I'm working on mark autocompletion now.

**Pull request merged**